weave-js/src/components/PagePanelComponents/Home/Browse3/smallRef/SmallRef.tsx

```diff
@@ -6,17 +6,32 @@ import {
 import React, {FC} from 'react';
 
 import {SmallArtifactRef} from './SmallArtifactRef';
-import {SmallWeaveRef} from './SmallWeaveRef';
+import {SmallOpVersionsRef} from './SmallOpVersionsRef';
+import {SmallObjectVersionsRef, SmallWeaveRef} from './SmallWeaveRef';
 
 export const SmallRef: FC<{
   objRef: ObjectRef;
   iconOnly?: boolean;
   noLink?: boolean;
 }> = ({objRef, iconOnly = false, noLink = false}) => {
   if (isWeaveObjectRef(objRef)) {
-    return (
-      <SmallWeaveRef objRef={objRef} iconOnly={iconOnly} noLink={noLink} />
-    );
+    if (objRef.artifactVersion === '*') {
```

Comment:
nit: I would move this switch logic to `SmallWeaveRef`. Given that these conditionals are specific only to the Weave-style refs. Other similar switching happens in `SmallWeaveRef`